lib/spack/llnl/util/cpu/microarchitectures.json

```diff
@@ -828,7 +863,8 @@
         "abm",
         "avx",
         "xop",
-        "lwp",
+        "xop",
```

Comment:
There seems to be a duplicate of `xop` here.

Reply:
Cut 'n' paste oversight. Will fix.